### ISSUE TITLE
The app is very hard to find via search engines and stores

### ISSUE THREAD

**Issue by wnhre2ur8cxx8** [LeastAuthority/destiny#244]
You chose a very generic name for your project. It is very hard to refer other people to "destiny" because there are so many different apps and programs with the same name. I  could not find it myself in the google appstore, I had to google it with "destiny wormhole" to get to the github site, and then click on the playstore link.

I have it already installed, but when I want to send some friend something it is too hard to tell them how to install it. 

I propose that you at least add some identifying words to the name. Like Destiny - Share your files, or i dont know :) with just the name destiny I dont know how one would (successfully) search for it. Just try it out yourselves: https://play.google.com/store/search?q=destiny&c=apps

**Comment by meejah** [comment]
Yes, I had this same problem :sweat_smile: trying to show a person at a meetup where to install it. Thanks for the report!

**Comment by wuan** [comment]
As of now searching for "destiny file transfer" has been quite useful after we have updated the App name to "Destiny – Secure File Transfer". We will try to improve this.

**Comment by trymeouteh** [comment]
An official website would be useful even if the official website is just an index page just like LocalSend does
https://localsend.org

Maybe these domains are available...

destiny.app
destinyapp.com
